private/buf/cmd/buf/buf.go

```diff
@@ -338,7 +342,7 @@ See https://docs.buf.build/faq for more details.`,
 					},
 				},
 			},
-			push.NewCommand("push", builder, "", false),
+			push.NewCommand("push", builder, pushDeprecationMessage, true),
```

Comment:
If we don't delete the commands altogether, then we have to maintain them forever. If we are going to move them now, we need to use `appcmd.NewDeletedCommand`, or don't delete them at all.